src/physics/linear_winds.f90

```diff
@@ -574,28 +576,50 @@ subroutine setup_remote_grids(u_grids, v_grids, terrain, nz)
 
     end subroutine setup_remote_grids
 
-    subroutine copy_data_to_remote(wind, grids, LUT, i,j,k, z)
+    subroutine copy_data_to_remote(wind, grids, LUT, LUT_co, LUT_index_co, i, j, k, z)
         implicit none
         real,           intent(in)  :: wind(:,:)
         type(grid_t),   intent(in)  :: grids(:)
-        real,           intent(inout):: LUT(:,:,:,:,:,:)[*]
-        integer,        intent(in)  :: i,j,k, z
-
-        integer :: img
-
+        real,           intent(inout):: LUT(:,:,:,:,:,:)
+        real,           intent(inout):: LUT_co(:,:,:)[*]
+        integer,        intent(inout):: LUT_index_co(:,:)[*]
+        integer,        intent(in)  :: i, j, k, z
+        integer :: img, LUT_i, LUT_j, LUT_k, LUT_z
+
+        ! each image will communicate the wind values it has calculated for an ijkz
+        ! location to the other images, the ijkz values are also sent so the receiving
+        ! images know where to put the incoming values
         do img = 1, num_images()
             associate(ims => grids(img)%ims, &
                       ime => grids(img)%ime, &
                       jms => grids(img)%jms, &
                       jme => grids(img)%jme  &
                 )
-            !$omp critical
```

Comment:
double check that `!$omp critical` isn't needed. Are there other OpenMP calls in the LUT routines